#All gains
kp_xPos: 20.0
ki_xPos: 0
kd_xPos: 0

kp_yPos: 50.0
ki_yPos: 2
kd_yPos: 5

kp_xHydro: 0.5e6
ki_xHydro: 0.0
kd_xHydro: 0.0

kp_yHydro: 0.5e6
ki_yHydro: 0.0
kd_yHydro: 0.0

kp_depth: 150.0
ki_depth: 4.0
kd_depth: 50 #100 #50.0 # (vservo) # 10.0 (normal)

kp_pitch: 1.0
ki_pitch: 0.0
kd_pitch: 0.0

kp_roll: 1.0
ki_roll: 0.0
kd_roll: 0.0

kp_yaw: 20.0
ki_yaw: 0.0
kd_yaw: 5.0

#Speed Coefficients, they don't really mean anything right now...

surge_coeff: 5
sway_coeff: 5

yaw_error_tol: 0.1  # radians
x_error_tol: 0.1  # m
y_error_tol: 0.1  # m
depth_error_tol: 0.05  #m
